libraries/abstractions/ble_hal/test/include/iot_test_ble_hal_afqp.h

```diff
@@ -33,137 +33,17 @@
 
 #include "iot_test_ble_hal_common.h"
 
-
-typedef enum
-{
-    bletestATTR_SRVCA_SERVICE,
-    bletestATTR_SRVCA_CHAR_A,
-    bletestATTR_SRVCA_NUMBER
-} bletestAttSrvA_t;
-
-
-typedef enum
-{
-    bletestATTR_SRVCB_SERVICE,
-    bletestATTR_SRVCB_CHAR_A,
-    bletestATTR_SRVCB_CHAR_B,
-    bletestATTR_SRVCB_CHAR_C,
-    bletestATTR_SRVCB_CHAR_D,
-    bletestATTR_SRVCB_CHAR_E,
-    bletestATTR_SRVCB_CCCD_E,
-    bletestATTR_SRVCB_CHAR_F,
-    bletestATTR_SRVCB_CCCD_F,
-    bletestATTR_SRVCB_CHARF_DESCR_A,
-    bletestATTR_SRVCB_CHARF_DESCR_B,
-    bletestATTR_SRVCB_CHARF_DESCR_C,
-    bletestATTR_SRVCB_CHARF_DESCR_D,
-    bletestATTR_INCLUDED_SERVICE,
-    bletestATTR_SRVCB_NUMBER
-} bletestAttSrvB_t;
-
 typedef struct
 {
     size_t xLength;
     uint8_t ucBuffer[ bletestsSTRINGYFIED_UUID_SIZE ];
 } response_t;
 
-static void prvSetGetProperty( BTProperty_t * pxProperty,
-                               bool bIsSet );
-static void prvStartAdvertisement( void );
+void prvBLESetUp( void );
```

Comment:
Do you need that function elsewhere? Should it be in the c file as a static?

Reply:
Let me explain the why the functions used in only one files are not static and why the function declarations in header files can be static here. Bora made the architecture that each c files includes their corresponding header file(the same name), and all the header files include the same common.h. since all the header files except for common.h are included by one file now, the function declarations inside can be static. I removed the "static" in latest commit so that they can be shared with others in the future. Currently they are exactly used in only one file, even can be moved out header file to c file. But if I move all things out, the header files will have no content at all.